src/Sentry/Internal/Hub.cs

```diff
@@ -258,6 +260,7 @@ public void Dispose()
                 }
             }
 
+            Interlocked.Exchange(ref _isEnabled, 0);
```

Comment:
It's just missing the check to avoid double dispose:

```suggestion
            if (Interlocked.Exchange(ref _isEnabled, 0) != 0)
            {
                return; // Already disposed
            }
```

Reply:
But this must be the first thing (maybe after logging)

Reply:
Oh, I see.